applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle the DVFS $(dvfs_state) state"
+	default y
+	# depends on $(dvfs_dependencies)
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "Timeout in milliseconds for the DVFS $(dvfs_state) state" if $(dvfs_timeout) != 0
```

Comment:
Are you sure that my suggestions really works?

After expansion, it looks like following for various DVFS states:

- `if 0 != 0`
- `if 5000 != 01

Reply:
Yes I checked that in .config file there is a correct value and that in menuconfig you can only change value for SMP and config_channel states